roles/generate-jenkins/templates/README.j2

```diff
@@ -211,7 +212,7 @@ services:
 
 ### docker cli
 
-```
+```bash
```

Comment:
This bash formatting does nothing to the cli samples except for formatting the "optional" directive incorrectly

Reply:
we could alternatively use `shell` instead of `bash` which will keep the linter happy and make sense logically across the files.

Reply:
Github formats optional fine imho. Switching to shell is a good point, although the endresult is the same 
![image](https://user-images.githubusercontent.com/7110194/113630221-ccff1880-9667-11eb-86c4-ef8f4ff990dc.png)